docs/content/misc/contribute.ngdoc

```diff
@@ -61,7 +61,8 @@ To build AngularJS, you clone the source code repository and use Grunt to genera
 minified AngularJS files:
 
 ```shell
-# Clone your Github repository:
+# Clone your Github repository(Note: Do not go with `sudo` user when cloning into this repository 
```

Comment:
I think it can't hurt to add the link to the sudo man page to the first mention of sudo on the page on line 49.